crypto/asn1/asn1_lib.c

```diff
@@ -114,9 +114,20 @@ OPENSSL_DECLARE_ERROR_REASON(ASN1, UNSUPPORTED_TYPE)
 
 static void asn1_put_length(unsigned char **pp, int length);
 
-int asn1_get_object_maybe_indefinite(const unsigned char **inp, long *out_len,
-                                     int *out_tag, int *out_class, long in_len,
-                                     int indefinite_ok) {
+// |ASN1_get_object| parses an ASN.1 header, including tag, class, and length
+// information. The tag number is written to |*out_tag|. The class is written to
+// |*out_class|. If the tag is not indefinite, the content length is written to
```

Comment:
nit: might want to document that this gets set to 0 if tag is indefinite.

Reply:
Added, thanks!